<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PushView</class>
 <widget class="QWidget" name="PushView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#PushView{
	background-color: white;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="layoutLeft">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="layoutTop">
       <item>
        <widget class="QLabel" name="labelProjNameText">
         <property name="text">
          <string>试卷名称:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelProjName">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layoutTableHead" stretch="1,1,7">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="labelTable">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QLabel{
	border: 2px solid rgb(26, 222, 209);
	color: #FFFFFF;
	background-color: rgb(26, 222, 209);
}</string>
           </property>
           <property name="text">
            <string>考生列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnImport">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: #FFFFFF;
	border: 2px solid rgb(24, 204, 192);
	background-color: rgb(20,196,188);
}
QPushButton:hover{
	background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
           </property>
           <property name="text">
            <string>导入</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horSpacerTable">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutInfo" stretch="0,0,0,1,0,0">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBoxTime">
       <property name="title">
        <string>答题时间</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="layoutTimeStart">
          <item>
           <widget class="QLabel" name="labelTimeStart">
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEditStart"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutTimeEnd">
          <item>
           <widget class="QLabel" name="labelTimeEnd">
            <property name="text">
             <string>结束</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEditEnd"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbbScoreInClient">
       <property name="text">
        <string>考生端改分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPush">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: #FFFFFF;
	border: 2px solid rgb(24, 204, 192);
	background-color: rgb(20,196,188);
}
QPushButton:hover{
	background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
       </property>
       <property name="text">
        <string>发放试卷</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNote">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1. 当考生列表确认，并且答题时间无误后，点击发放试卷，开放考生端连接。&lt;/p&gt;&lt;p&gt;2. 到达答题开始时间时，程序自动开放考生端作答，考生开始答题。&lt;/p&gt;&lt;p&gt;3. 到达答题结束时间时，程序自动结束考生端作答，回收试卷。&lt;/p&gt;&lt;p&gt;考生答题数据会同步显示在考生列表中。&lt;/p&gt;&lt;p&gt;若勾选“考生端改分”，考试结束后，在教师端改分的同时，答案也会下发到考生端，让考生端显示改分情况和各题情况。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verSpacerNote">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: #FFFFFF;
	border: 2px solid rgb(24, 204, 192);
	background-color: rgb(20,196,188);
}
QPushButton:hover{
	background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
